src/keyboardObject.js

```diff
@@ -0,0 +1,921 @@
+import {backspaceButtonAction, tabButtonAction, deleteButtonAction, capsLockButtonClickAction, enterButtonAction,
+    commonShiftHandler, upButtonAction, commonAltHandler, leftButtonAction, downButtonAction, rightButtonAction,
+    spaceButtonAction} from "./index.js";
+
+const keyboardElements = [
+    {
```

Comment:
It's all good but there's a better structure way. Instead of using an array here, you can use an object with keys = keycods. This way you won't have to search keyboard element object by keyCode, but just pick it by key (keyboardElements[keyCode]. Example: 

keyboardElements = {192: {values.... etc.

Reply:
You don't have to change it, just FYI